src/components/BusinessCaseReview/index.test.tsx

```diff
@@ -375,7 +378,7 @@ describe('The Business Case Review Component', () => {
       },
       hasUserInterface: 'YES'
     }
-  };
+  } as BusinessCaseModel;
```

Comment:
Kind of a nitpick, but typecasting always makes me a bit nervous. The root cause of the type error without the cast is that `businessCase.status` gets widened to the type `string` instead of the union of string literals, `type BusinessCaseStatus = 'OPEN' | 'CLOSED';`.

Two possible solutions to make it type-check:
1. Add a type annotation to the declaration of `businessCase`, i.e. `const businessCase: BusinessCaseModel = ...`. This will still throw a type error if the value assigned to `businessCase` isn't a valid `BusinessCaseModel, i.e. if you have `status: 'NOTAVALIDSTATUS'`, but it indicates that `businessCase` can be passed as a prop to the `<BusinessCaseReview>` component.
2. Replace the typecast with `as const`, a [const assertion](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions), which keeps `businessCase.status` from being widened.